README.md

```diff
@@ -0,0 +1 @@
+docs/README.md
```

Comment:
Re: Readme... The doc generator looks in docs, but github looks in root.  A link seemed like a reasonable compromise.